### PR TITLE
LogSumExp

### PR DESCRIPTION
**Pull request by cscherrer**
Hi @joshday , I came across [this](https://github.com/JuliaStats/LogExpFunctions.jl/blob/e5cc905abcc0bedf706f2cdc68e1a43cb59f29ce/src/logsumexp.jl#L26) and it hit me that an OnlineStats version should be pretty simple. I think it works out nicely, let me know what you think :)

### CODE REVIEW

File: src/OnlineStats.jl [hunk 1]
```diff
@@ -47,7 +47,7 @@ export
     OrderStats,
     Part, Partition, PlotNN, ProbMap, P2Quantile,
     Quantile,
-    ReservoirSample,
+    ReservoirSample, LogSumExp,
```
Comment (joshday) [inline src/OnlineStats.jl:50]: Exports are alphabetical.  That's my only (very minor) comment for this PR

**Comment by joshday** [comment]
Sweet!  Thanks for the contribution!  I made a minor comment.  If you don't have time I'll merge it myself and change it.

File: src/OnlineStats.jl (hunk 1 above)
Comment (cscherrer) [reply src/OnlineStats.jl:50]: Ah, ok I didn't notice that. Should be a quick fix :)

**Comment by cscherrer** [comment]
Fixed! 🙂

**Pull request merged**